content/riak/kv/2.2.0/using/repair-recovery/repairs.md

```diff
@@ -91,50 +101,135 @@ If there are compaction errors in any of your vnodes, those will be listed in th
 ./442446784738847563128068650529343492278651453440/LOG 
 ```
 
-<div class="note">
-<div class="title">Note</div>
+{{% note %}}
 While corruption on one vnode is not uncommon, corruption in several vnodes very likely means that there is a deeper problem that needs to be address, perhaps on the OS or hardware level.
-</div>
+{{% /note %}}
+
 
 ## Healing Corrupted LevelDBs
 
-The first step in properly addressing this problem is to stop the node.
+When you have discovered corruption in your LevelDB backend, the steps you take to resolve it will depend on whether you are using [tiered storage] or not.
+
+Choose your setup below: 
+
+1. [Just LevelDB](#leveldb)
+2. [LevelDB with tiered storage](#leveldb-with-tiered-storage)
+
+
+### LevelDB
+
+Follow the steps below to heal your corrupted LevelDB.
+
+1\. Stop the node:
 
 ```bash
 riak stop
 ```
 
-Repairing the corrupted LevelDB can be done through the [Erlang shell](http://learnyousomeerlang.com/starting-out). Do not start Riak at this point; use the shell only.
+2\. To repair the corrupted LevelDB through the [Erlang shell],  you will run the the `riak ertspath` command to output the path to Riak's internal Erlang runtime, and the `erl` command to start the Erlang shell. You can run them in a single command: 
+
+```bash
+`riak ertspath`/erl
+```
+
+{{% note title="Erlang version" %}}
+Note, you must start up the Erlang shell using the same version of Erlang packaged with Riak. The above command will make sure you do so. If you choose not to use the above command please pay close attention to the version and location you use with the `erl` command.
+{{% /note %}}
+
+3\. Once in the shell, run the following command:
 
-You can fire up the shell by running the `erl` command. To ensure that you start up the shell using the same version of Erlang that's embedded with Riak, you should run the `erl` command as an absolute path. Here's an example:
+```erlang
+application:set_env(eleveldb, data_root, "").
+```
+
+4\. Then set `Options` equal to an empty list:
+
+```erlang
+Options = [].
+```
+5\. For each corrupted LevelDB that you found using the [`find` command above](#checking-for-compaction-errors), run the following `repair` command, substituting the path to your LevelDB vnodes and the appropriate vnode number:
+
+```erlang
+DataRoot = "»path to your data root«".
+VNodeNumber = "»vnode id you want to repair«".
+RepairPath = DataRoot ++ "/" ++ VNodeNumber.
+eleveldb:repair(RepairPath, Options).
+```
```

Comment:
If the customer has multiple VNode ID's to repair, assigning the new VNodeNumber will cause a no match issue.  If a customer isn't comfortable with Erlang, this may cause some confusion.  Perhaps just using the one-liner approach below would be best unless we add  a section on additional partitions and using f(VNodeNumber) and f(RepairPath) to allow additional repair assignments?